std_msgs/Header header
int64 traj_vel                       # traj_vel reading. 0 is interpreted as false

